samples/bluetooth/peripheral_power_profiling/src/pwr_service.h

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef PWR_SERVICE_H_
+#define PWR_SERVICE_H_
+
+#include <zephyr/bluetooth/conn.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @brief PWR Service event. */
```

Comment:
Service or service? You are using both in this file.

Reply:
Lets use Service in this file